README.md

```diff
@@ -33,8 +33,8 @@ The package can be installed as:
     adapter: Bamboo.SMTPAdapter,
     server: "smtp.domain",
     port: 1025,
-    username: "your.name@your.domain",
-    password: "pa55word",
+    username: "your.name@your.domain", # or {:system, "SMTP_USERNAME"}
+    password: "pa55word", # or {:system, "SMTP_PASSWORD"}
```

Comment:
Could you use `"SMTP_PASS"` here? Or, update `"SMTP_PASS"` in the [library documentation](https://github.com/fewlinesco/bamboo_smtp/blob/develop/lib/bamboo/adapters/smtp_adapter.ex#L16) to use `"SMTP_PASSWORD"`?

We can use `SMTP_PASSWORD`, the naming is better but we would just like to keep all the example consistent. :)